dealer.go

```diff
@@ -51,6 +52,11 @@ func (dealer *dealerSocket) Type() SocketType {
 	return dealer.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.